docs/en/observability/observability-ui.asciidoc

```diff
@@ -27,42 +27,41 @@ Additionally, the alerts chart keeps you informed of any issues that you may nee
 to resolve quickly.
 
 [float]
-[[view-services-and-transactions]]
-== Services and transactions
-
-The *APM* chart displays the total number of services running within your environment
-and the total number of transactions per minute that were captured by the Elastic APM
-agent instrumenting those services.
+[[view-alerts]]
+== Alerts
 
-To drill-down and view this APM data in the *APM app*, click *View in app*.
+To help keep you aware of potential issues in your environments, the *Alerts* feed
+provides a snapshot of the alerts occurring within the specified time frame. Displayed are the
```

Comment:
Depends on earlier comment but reads awkwardly to me - better to start 'This view displays the rule types...' or similar?

Reply:
Ah, I agree. This was existing text -- I just moved it up from the bottom of the page to reflect the order of charts in the UI. What about something like:

>The **Alerts** table provides a snapshot of alerts occurring within the specified time frame.
The table includes the alert status, when it was last updated, the reason for the alert, and more.
>
>To see more details about these alerts, click **Show alerts**. For more information,
see [Create rules](https://www.elastic.co/guide/en/observability/current/create-alerts.html).